ui/src/components/SubDescription/SubDescription.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+import styled from 'styled-components';
+import { z } from 'zod';
+import { variables } from '@splunk/themes';
+import { SubDescriptionSchema } from '../../types/globalConfig/pages';
+
+export const SubTitleComponent = styled.div.attrs({
+    className: 'pageSubDescription',
```

Comment:
what the purpose of having class instead of direct styles assigning?

Reply:
wanted to keep the structure similar to title etc. but tru actually not needed, removed